SETUP/upgrade/16/20210802_clean_user_profiles.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
Please rename this script `20210802_alter_user_profiles.php` since we're altering the table and not removing rows from it. The script's contents look good to me.

Reply:
Done this.